### DISCUSSION TITLE
Welcome to SequencePlugin Discussions!

### DISCUSSION THREAD

**Discussion by Vanco** [Vanco/SequencePlugin#68] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jderijke** [comment]
Absolutely great plugin. 
The navigation feature is awesome.
1 small feature request:
- the method return arrows should have dashed lines. They can also be omitted (my personal preference as it makes the diagram lighter and the end of the activation bar is sufficient indication of the end of the method)
Keep up the great work!

**Reply by Vanco** [reply]
There is an `Show return arrows` option in tools > Sequence Diagram, you need to option out.

**Reply by jderijke** [reply]
return arrows are fine now, thanks!

**Comment by dresh00** [comment]
Hi, can I export this in a format that can be used in another tool where I can manipulate the sequence and flow generated? 
Can we export the generated sequence diagram from this tool in a UML format, for example?

**Reply by dresh00** [reply]
looking for web/macOS

**Reply by Vanco** [reply]
The PlantUML and Mermaid format supported.

**Comment by alosireply** [comment]
@Vanco 
I use the tool a lot and it's very interesting. Would it be feasible to add the option to print the http requests that are made instead of the methods on the sequence arrows (both in input to the java application and in output)?

**Comment by Esaxiya** [comment]
How should I use the export image function and where is the entry？

**Reply by Vanco** [reply]
First, you should generate one Sequence Diagram.  
When Diagram shows in tool windows, there are some small icon buttons, one of those is `Export image...`，Click it, and in the save file dialog, choose the image file type, then click OK.    The image will generated in the directory.
Have fun!
